src/ch/jamiete/hilda/Hilda.java

```diff
@@ -143,7 +119,7 @@ public void start() {
         this.executor.setRemoveOnCancelPolicy(true);
         this.executor.setMaximumPoolSize(10);
 
-        final long rotate = this.getNextMidnightInMillis("GMT+10") - System.currentTimeMillis();
+        final long rotate = Util.getNextMidnightInMillis("GMT+10") - System.currentTimeMillis();
```

Comment:
This should probably be in a separate commit

Reply:
Saves time. Multiple commit clutters git log,

Reply:
The git log will survive another day.